nautobot_floor_plan/templates/nautobot_floor_plan/floorplantile_retrieve.html

```diff
@@ -49,6 +49,14 @@
                 <td>Rack Group</td>
                 <td>{{ object.rack_group| hyperlinked_object }}</td>
             </tr>
+            <tr>
+                <td>Tenant</td>
+                <td>{{ object.rack.tenant | hyperlinked_object }}</td>
+            </tr>
+            <tr>
+                <td>Tenant Group</td>
+                <td>{{ object.rack.tenant.tenant_group | hyperlinked_object }}</td>
```

Comment:
```suggestion
                <td>{{ object.rack.tenant.tenant_group | hyperlinked_object | placeholder }}</td>
```
If a Rack doesn't have a `.tenant`, it fails to return the placeholder for the `.tenant.tenant_group`, so this just adds it back in.